packages/pipeline/migrations/1550163069315-TokenOrderBookSnapshotsAddMakerAddress.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+const TOKEN_ORDERBOOK_SNAPSHOT_TABLE = 'raw.token_orderbook_snapshots';
+const NEW_COLUMN_NAME = 'maker_address';
+
+export class TokenOrderBookSnapshotsAddMakerAddress1550163069315 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        const snapshotTable = await queryRunner.getTable(TOKEN_ORDERBOOK_SNAPSHOT_TABLE);
+        if (snapshotTable) {
```

Comment:
Why this `if`?  Shouldn't it be presumed that when this migration is run the table *will* exist?  (This isn't necessarily a critique; it's just as much for my own edification.)  In any case, if the `if` really is useful, then maybe add an error/warning/info message in an `else` block?

Reply:
The `if` is there because typescript requires it. Adding throws.